.github/actions/tests/k8s-e2e-tests/action.yml

```diff
@@ -73,6 +73,27 @@ runs:
         kubectl config use-context my-context
       shell: bash
 
+    - name: Copy all decrypted keys from binary-host pod
```

Comment:
We want to remove `binary-host` completely, and we need secrets to be available by `partner-chains-node` binary so we need copy them into the pod directly. The best way to do that is leveraging existing pytest fixtures, since they allow us to clean them up after tests are done.